.vib/gitlab-runner/goss/gitlab-runner.yaml

```diff
@@ -0,0 +1,6 @@
+file:
+  /opt/bitnami/gitlab-runner/bin/gitlab-runner:
+    exists: true
+    filetype: file
+    contains:
+      - "arm64"
```

Comment:
This file is a **binary**. Checking for the presence of a string that comes from the compilation stage in files of this type is not a good practice (or simply, checking strings in binary files in most cases). I'd remove this.

Reply:
Besides, the binary does not seem to include the string. The tests do pass because you forgot to include this file in the `goss.yaml` general file.